code-scanning/properties/black-duck-security-scan-ci.properties.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Black Duck Security Scan Workflow",
+    "creator": "Black Duck Software, Inc.",
+    "description": "The Black Duck Security Scan GitHub Action allows you to configure your pipeline to run Black Duck Security Scan and take action on the security results",
+    "iconName": "black-duck.svg",
```

Comment:
Hi @sadmananik it looks like we need to rename "black-duck.svg" to "black-duck" for the the CI test to pass

Reply:
Thanks @AlexDeMichieli.  Done.